tools/scripts/platform/xilinx/build_project.tcl

```diff
@@ -5,6 +5,6 @@ if { $argc != 1 } {
 
 sdk setws [lindex $argv 0]
 
-projects -build -name app -type app
+app build -name app
```

Comment:
I would either make a visible note about this change in the commit description or place it in a separate commit. It probably is best if placed in a separate commit because, if I'm not mistaken, this it an update to the make process following an update to the Xilinx tools. I'd say it should special notice.

Reply:
Yes, this is true. It is the current recommended (by the Xilinx tools) way of dealing with projects.